.github/workflows/maven.yaml

```diff
@@ -28,6 +28,7 @@ on:
     branches: [ '*' ]
   pull_request:
     branches: [ '*' ]
+  workflow_dispatch:
```

Comment:
Why add this?

Reply:
Sorry, it wasn't intended to go into this PR. I wanted to setup Github Actions on my fork of fluo so I could re-run the workflow and see if the IT failed consistently. I shouldn't have committed this change to the branch the PR was setup for. I will back it out and test on a different branch.

Reply:
Ah, that makes sense.